pymc3/step_methods/metropolis.py

```diff
@@ -818,6 +880,294 @@ def competence(var, has_grad):
+                                                      **{"is_mlda_base": True})
+        else:
+            # drop the last coarse model
+            next_coarse_models = self.coarse_models[:-1]
+            with self.next_model:
+                # make sure the correct variables are selected from next_model
+                vars_next = [var for var in self.next_model.vars
+                             if var.name in self.var_names]
+                # MLDA sampler in some intermediate level, targeting self.next_model
+                self.next_step_method = pm.MLDA(vars=vars_next, base_S=self.base_S,
+                                                base_proposal_dist=self.base_proposal_dist,
+                                                base_scaling=self.base_scaling,
+                                                tune=self.tune,
+                                                base_tune_interval=self.base_tune_interval,
+                                                model=None, mode=self.mode,
+                                                subsampling_rate=self.subsampling_rate,
+                                                coarse_models=next_coarse_models,
+                                                base_blocked=self.base_blocked,
+                                                **kwargs)
+
+        # instantiate the recursive DA proposal.
+        # this is the main proposal used for
+        # all levels (Recursive Delayed Acceptance)
+        # (except for level 0 where the step method is Metropolis and not MLDA)
+        self.proposal_dist = RecursiveDAProposal(self.next_step_method,
+                                                 self.next_model,
+                                                 self.tune,
+                                                 self.subsampling_rate)
+
+        # Update stats data types dictionary given vars and base_blocked
+        if self.base_blocked or len(self.vars) == 1:
+            self.stats_dtypes[0]['base_scaling'] = np.float64
+        else:
+            for name in self.var_names:
+                self.stats_dtypes[0]['base_scaling_' + name] = np.float64
+
+    def astep(self, q0):
+        """One MLDA step, given current sample q0"""
+        # Check if the tuning flag has been changed and if yes,
+        # change the proposal's tuning flag and reset self.accepted
+        # This is triggered by iter_sample while the highest-level MLDA step
+        # method is running. It then propagates to all levels.
+        if self.proposal_dist.tune != self.tune:
+            self.proposal_dist.tune = self.tune
+            # set tune in sub-methods of compound stepper explicitly because
+            # it is not set within sample() (only the CompoundStep's tune flag is)
+            if isinstance(self.next_step_method, CompoundStep):
+                for method in self.next_step_method.methods:
+                    method.tune = self.tune
+            self.accepted = 0
```

Comment:
Here, if the bottom-level Metropolis methods are within a Compound step, I manually set their tune flags 

CompoundStep has a stop_tuning() function that is called by _iter_sample to set tune=False for all the methods contained in it. But the Metropolis class does not have a stop_tuning() function thus the metropolis flags are not set. 

If there is no reason not to, should I add a stop_tuning() to Metropolis?